test/distributed/fsdp/test_checkpoint_wrapper.py

```diff
@@ -184,12 +187,12 @@ def test(use_checkpointing, use_wrapper, use_reentrant):
                 use_checkpointing,
                 use_wrapper=use_wrapper,
                 use_reentrant=use_reentrant,
-            ).cuda()
-            x = torch.randn(10000, 256, requires_grad=True).cuda()
-            torch.cuda.reset_peak_memory_stats()
+            ).to(device_type.type)
+            x = torch.randn(10000, 256, requires_grad=True).to(device_type.type)
+            torch.get_device_module(device_type.type).reset_peak_memory_stats()
```

Comment:
Because you remove `@unittest.skipIf(not torch.cuda.is_available(), "Test requires CUDA")`, this line should fail when running on only CPU devices. I got the `module 'torch.cpu' has no attribute 'reset_peak_memory_stats'` error with the internal CPU tests. I'm not sure why CI didn't fail though. cc., @clee2000

Reply:
It is disabled in CI due to https://github.com/pytorch/pytorch/issues/79510

Reply:
I see. So we should also disable this internally.

Reply:
@rahulsingh-intel Can you add an unconditional skip to this test? Thanks!

Reply:
hi @fegin Added.
Thanks.